packages/bpk-component-content-cards/src/BpkContentCards.module.scss

```diff
@@ -23,17 +23,18 @@
 @use '../../unstable__bpk-mixins/breakpoints';
 @use '../../unstable__bpk-mixins/typography';
 
-.bpk-content-cards {
-  &--header-text {
-    margin-bottom: tokens.bpk-spacing-base();
+.headerText {
+  margin-bottom: tokens.bpk-spacing-base();
```

Comment:
Does this `margin-bottom` not get overridden by `@include typography.bpk-text;`?

Reply:
Ah, without it, the image and content are no longer aligned as they were previously, which means the Percy snapshot match will fail:

With:
![Screenshot 2024-03-25 at 09 36 55](https://github.com/Skyscanner/backpack/assets/7976511/5f493b60-06ff-446a-9b41-a2f6652d0c14)

Without:
![Screenshot 2024-03-25 at 09 40 10](https://github.com/Skyscanner/backpack/assets/7976511/4771b96a-a499-43ea-9129-abcced38b8cd)

Reply:
Doesn't override, cool 👍

Reply:
Ah, I need to look back into this some user error on my side with branches - I need to check the above again.

Reply:
Cut a long story short:

Reply:
`bpk-paragraph` - https://github.com/Skyscanner/backpack/blob/main/packages/bpk-mixins/_typography.scss#L663
```
@mixin bpk-paragraph {
  margin-top: $bpk-spacing-none;
  margin-bottom: bpk-spacing-base();
}
```

**versus**

`bpk-text` https://github.com/Skyscanner/backpack/blob/main/packages/bpk-mixins/_typography.scss#L64
```
@mixin bpk-text {
  margin: 0;
}
```

Reply:
`margin: 0` isn't enough, we need that `margin-bottom` to be set to `bpk-spacing-base()` to match the alignment on `main`.

**I've also added some Visual Test stories**, we can reduce these though.

Reply:
Thanks Jim, LGTM. On the VT, my take is that we don't need all of them. My bias would be to the more complex example 3 cards & have that be the only VT for the component. Just thinking purely of extra cost on Percy tbh.

Reply:
Ah cool, I'll leave the three card Visual Test and remove the others 👍